src/llama-scripts/DistributeRewardsV2Script.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+import {IRegistryDiamond} from "src/interfaces/IRegistryDiamond.sol";
+import {LlamaAccount} from "src/accounts/LlamaAccount.sol";
+import {LlamaBaseScript} from "src/llama-scripts/LlamaBaseScript.sol";
+import {IERC20} from "@openzeppelin/token/ERC20/IERC20.sol";
+
+/// @title Distribute Rewards V2 Script
+contract DistributeRewardsV2ScriptBase is LlamaBaseScript {
+  /// @notice The `RegistryDiamond` contract address.
+  IRegistryDiamond public constant REGISTRY_DIAMOND = IRegistryDiamond(0x7c0422b31401C936172C897802CF0373B35B7698);
+
+  /// @notice The Treasury Llama account address.
+  LlamaAccount public constant RIVER_TREASURY = LlamaAccount(payable(0x8ee48C016b932A69779A25133b53F0fFf66C85C0));
+
+  /// @notice The RVR ERC20 token address.
+  IERC20 public constant RVR_TOKEN = IERC20(0x91930fd11ABAa5241241d3B07c02A8d0B5ac1920);
+
+  /// @notice The period distribution amount.
+  uint256 public constant PERIOD_DISTRIBUTION_AMOUNT = 30_769_230_769_200_000_000_000_000;
```

Comment:
By hard coding this to make a change to the period amount we would need to:

1. update this script
2. redeploy the updated script
3. gov action to depermission the old script
4. gov action to permission the new script
5. pr against llama instance to update new script address

Do you see that there are multiple steps needed to make a small change to period amount ? If we read/set the period amount from governance contracts as we did previously, we would not have to undertake the above.

Reply:
Addressed.